packages/vue/src/components/molecules/SfRadio/SfRadio.vue

```diff
@@ -93,6 +93,7 @@ export default {
   methods: {
     inputHandler() {
       this.$emit("input", this.value);
+      this.$emit("change", this.value);
```

Comment:
why do we want two events to be emitted at the same time? If it is about the input event not supported, then this is not the right place to add the emitter. This `inputHandler` is tied to a specific event listener of the radio button `@input="inputHandler"` which is the reported problem. It should be `@change="changeHandler`

Reply:
Yes, you're right, I changed that.